packages/@aws-cdk/aws-iam/lib/role.ts

```diff
@@ -190,6 +209,11 @@ export class Role extends Resource implements IRole {
    */
   public readonly roleName: string;
 
+  /**
+   * Returns the permissions boundary of the role.
+   */
```

Comment:
We normally don’t re-expose configuration at the interface level since it would require anyone who imports a role to now specify this. Is there a concrete use case for exposing?

Reply:
Makes sense. I'll remove it.